CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 3.92.2 - TBD
```

Comment:
Refer to the instructions for adding a changelog here: [CONTRIBUTING - changelog](https://github.com/aws/aws-sdk-php/blob/master/CONTRIBUTING.md#changelog-documents).

This will get added to CHANGELOG.md automatically in the right release.

Reply:
That makes sense! Fixed, thanks :)